<html>
<head><meta charset="utf-8"><title>ValueError when installing my own package · python-questions · Zulip Chat Archive</title></head>
<h2>Stream: <a href="http://127.0.0.1:4000/html/stream/10-python-questions/index.html">python-questions</a></h2>
<h3>Topic: <a href="http://127.0.0.1:4000/html/stream/10-python-questions/topic/ValueError.20when.20installing.20my.20own.20package.html">ValueError when installing my own package</a></h3>

<hr>


<head><link href="http://127.0.0.1:4000/style.css" rel="stylesheet"></head>

<a name="22174"></a>
<h4><a href="https://zulip2.cloud.ucar.edu#narrow/stream/10-python-questions/topic/ValueError%20when%20installing%20my%20own%20package/near/22174" class="zl"><img src="../../../assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Isla Simpson <a href="http://127.0.0.1:4000/html/stream/10-python-questions/topic/ValueError.20when.20installing.20my.20own.20package.html#22174">(Nov 27 2020 at 15:45)</a>:</h4>
<p>Hello, my google skills are failing me on this.  I have made my own package containing functions I want to use.  I have also made my own environment because I wanted to become familiar with that.  I have the same environment on cheyenne and the cgd machines and I can successfully install my package on the cgd machines using "pip install -e . --user".  I can also successfully install it on cheyenne in the base environment but when I try to do it within my own environment, I get an error ValueError: source code string cannot contain null bytes.  I can send along the longer error trace if that's helpful.</p>
<p>Googling suggests to me there is some kind of corrupt file somewhere.  I have cleared the pip cache but that didn't help.  Does anyone know what to do?</p>
<p>I am working here /home/islas/python/ecpaper2020/</p>
<p>and trying to install ecpaper_utils.</p>
<p>Thanks in advance for any help!</p>



<a name="22175"></a>
<h4><a href="https://zulip2.cloud.ucar.edu#narrow/stream/10-python-questions/topic/ValueError%20when%20installing%20my%20own%20package/near/22175" class="zl"><img src="../../../assets/img/zulip.svg" alt="view this post on Zulip" style="width:20px;height:20px;"></a> Isla Simpson <a href="http://127.0.0.1:4000/html/stream/10-python-questions/topic/ValueError.20when.20installing.20my.20own.20package.html#22175">(Nov 27 2020 at 16:14)</a>:</h4>
<p>Never mind, I have solved my problem.  I removed the environment and re-installed it and then it worked.  If anyone understands why this would have happened, I'd be glad to know.  But if not, it doesn't matter, it's all good now.  Thanks!</p>



<hr><p>Last updated: May 16 2025 at 17:14 UTC</p>
</html>